File: .ProgramingBasics/.ComplexConditionalStatements/01.PersonalTitles/src/PersonalTitles.java
import java.util.Scanner;

public class PersonalTitles {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double age = Double.parseDouble(scanner.nextLine());
        String sex = scanner.nextLine();

        if (sex.equals("m")) {
            if (age >= 16) {
                System.out.println("Mr.");
            }
            else if (age < 16) {
                System.out.println("Master");
            }
        }
        else if (sex.equals("f")) {
            if (age >= 16) {
                System.out.println("Ms.");
            }
            else if (age < 16) {
                System.out.println("Miss");
            }
        }
    }
}

File: ProgramingBasics/ComplexConditionalStatements/01.PersonalTitles/01.PersonalTitles/PersonalTitles.cs
using System;

namespace _01.PersonalTitles
{
    public class PersonalTitles
    {
        public static void Main(string[] args)
        {
            var age = double.Parse(Console.ReadLine());
            var gender = Console.ReadLine();

            if (gender =="m")
            {
                if (age >= 16)
                {
                    Console.WriteLine("Mr.");
                }
                else if (age < 16)
                {
                    Console.WriteLine("Master");
                }
            }
            else if (gender == "f")
            {
                if (age >= 16)
                {
                    Console.WriteLine("Ms.");
                }
                else if (age < 16)
                {
                    Console.WriteLine("Miss");
                }
            }
        }
    }
}